keybase/report_test.go

```diff
@@ -77,3 +77,18 @@ func TestReportActionEmpty(t *testing.T) {
 	err := ctx.reportAction("", updater.UpdateOptions{}, server.URL, 5*time.Millisecond)
 	assert.NoError(t, err)
 }
+
+func TestReportSuccess(t *testing.T) {
+	server := newServer("{}")
+	defer server.Close()
+
+	tmp := defaultEndpoints.success
+	defaultEndpoints.success = server.URL
+	defer func() { defaultEndpoints.success = tmp }()
+
+	ctx := testContext(t)
+	ctx.ReportSuccess(testOptions)
```

Comment:
Technically this improves coverage, but we can't validate the result of this exported call, and we _do_ validate its effective result below. So I'm not sure it's useful.